api/Blob.json

```diff
@@ -41,7 +41,7 @@
             "version_added": null
           },
           "webview_android": {
-            "version_added": null
+            "version_added": true
```

Comment:
This is Blob, https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/fileapi/blob.idl has no conditional exposure, so this is supported by WebView for sure.